src/tools/cephfs/JournalTool.cc

```diff
@@ -250,17 +257,36 @@ int JournalTool::main_journal(std::vector<const char*> &argv)
     }
   } else if (command == "reset") {
     bool force = false;
-    if (argv.size() == 2) {
+    if (argv.size() == 1) {
+        std::cerr << "warning: this operation resets the journal!!!\n"
```

Comment:
These messages should be more explicit. Something like "Do not run this operation if you do not understand CephFS' internal storage mechanisms or have received specific instructions from those who do"

Reply:
fixed